_Week9/Group4/estimation.do

```diff
@@ -0,0 +1,99 @@
+********************************************************************************
+* ECON 491
+* Week 9 - Estimation
+* Group 4
+* November 14, 2022
+********************************************************************************
+
+cap prog drop projectdata_sim
+
+prog define projectdata_sim, rclass
+
+clear
+
+********************************************************************************
+* setting up inputs
+********************************************************************************
+
+{
+	
+syntax anything
+local bias: word 1 of `anything' //a dummy for whether the results are biased or not
+local cluster_size: word 2 of `anything' //inputting cluster size
```

Comment:
I wonder whether you'd want to introduce some randomization in terms of cluster size, given that not all schools will be exactly the same size.